packages/toolkit/src/query/defaultSerializeQueryArgs.ts

```diff
@@ -27,6 +29,12 @@ export const defaultSerializeQueryArgs: SerializeQueryArgs<any> = ({
             }, {})
         : value,
     )
+    } catch (e) {
```

Comment:
(I'm relatively new to open source process, so if I should keep my comments to the issue rather than the PR, please let me know)

I suspect that for most use cases, silently swallowing an error is worse than throwing it.

Reply:
I think most likely you are correct, that's why I requested guidance on the error handling. Do you have any suggestions?

Reply:
Actually, I have a thought! Let me put together a PR.

Reply:
See: https://github.com/reduxjs/redux-toolkit/pull/4315/files